sandbox-server/app.js

```diff
@@ -25,4 +25,36 @@ app.post('/things', (request, response) => {
   response.send('New thing created: '+request.body)
 })
 
-app.listen(3000)
\ No newline at end of file
+app.get('/somefile', (request, response) => {
+  response.status(200)
+  let requestAcceptTypes = {
+    'text/plain': {
+      'method': 'send',
+      'responseMessage': 'This is a plain text file'
+    },
+    'text/html': {
+      'method': 'send',
+      'responseMessage': '<!DOCTYPE html><html><body>This is an HTML file</body></html>'
+    },
+    'application/json': {
+      'method': 'json',
+      'responseMessage': { "title": "some JSON data" }
+    }
+  }
+
+  for( let acceptType in requestAcceptTypes ) {
+    let values = requestAcceptTypes[acceptType]
+    if(request.accepts(acceptType)) {
+      response.type(acceptType)
+      response[values.method](values.responseMessage)
+    }
+  }
+})
+
+app.get('/old-page', (request, response) => {
+  response.status(301)
+  response.setHeader( 'Location', 'http://localhost:3000/newpage')
+  response.send()
+})
+
+app.listen(3000)
```

Comment:
this code looks the same as previous PRs that I have already commented on. Dont do the meta hash thing. It's confusing. Also just use `response.redirect('/newpage')`